en/docs/tutorials/create-your-first-rest-api.md

```diff
@@ -75,15 +79,13 @@ In this tutorial, let's design the REST API by updating the low-code diagram as
 
     ![Add resource](../assets/img/tutorials/rest-api/add-resource.png){.cInlineImage-half}
 
-    Then click **Resource**.
+    As a result, a pane named **Configure Resource** appears on the right of the page. In this pane, enter information as follows:
```

Comment:
This should be step 4 right?

Reply:
The tutorial is organised to have one step per statement/resource to minimize the step count. This step does not have sub-steps because there are not many actions related to the resource configuration.